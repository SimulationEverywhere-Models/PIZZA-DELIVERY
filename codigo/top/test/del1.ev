00:00:00:004 Llamada_del 3
00:00:00:008 Llamada_del 4
00:00:08:000 Abrir_del 1
00:00:13:000 Llamada_del 1
00:00:20:000 Llamada_del 5
00:00:25:000 Llamada_del 6
00:00:28:000 Llamada_del 7
00:00:50:000 Llamada_del 8
00:00:55:000 Llamada_del 9
00:00:57:000 Llamada_del 2
00:00:59:000 Llamada_del 10
