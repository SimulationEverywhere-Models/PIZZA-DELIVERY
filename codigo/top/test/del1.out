00:00:00:008 ocupada_del 1
00:00:00:019 no_atiende_del 1
00:00:15:000 error_del 1
00:00:35:120 salida_del 6
00:00:35:450 salida_del 5
00:00:51:090 salida_del 7
00:00:57:000 error_del 1
00:01:12:090 salida_del 8
00:01:20:090 salida_del 2
00:01:20:240 salida_del 10
